assets/common/src/register.py

```diff
@@ -211,7 +212,9 @@ def main():
     }
     json_object = json.dumps(model_info, indent=4)
 
-    with open(registration_details, "w") as outfile:
+    registration_file =  os.path.join(registration_details, "model_registration_details.json")
+
+    with open(registration_file, "w") as outfile:
```

Comment:
As discussed, please add this in try-except block to make sure component does not fail for any issue at this point

Reply:
Done